packages/x-charts/src/BarChart/extremums.test.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+import { getExtremumX } from './extremums';
+import { ExtremumGetter } from '../models/seriesType/config';
+
+const buildData = (
+  data: number[],
+  layout: 'vertical' | 'horizontal' = 'vertical',
+): Parameters<ExtremumGetter<'bar'>>[0] => {
+  return {
+    series: {
+      id1: {
+        id: 'id1',
+        type: 'bar',
+        color: 'red',
+        data,
+        stackedData: data.length
+          ? [
+              [data[0], data[1]],
+              [data[2], data[3]],
+            ]
+          : [],
+        layout,
+      },
+    },
+    axis: {
+      id: 'id',
+      data,
+    },
+    isDefaultAxis: true,
+  };
+};
+
+describe('BarChart - extremums', () => {
```

Comment:
Time will tell if this test isn't too low level. Testing public APIs tends to scale better.